containers/ecr-viewer/src/app/api/health-check/route.ts

```diff
@@ -1,9 +1,26 @@
 import { NextResponse } from "next/server";
 
+import { azureBlobStorageHealthCheck } from "@/app/data/blobStorage/azureClient";
+import { s3HealthCheck } from "@/app/data/blobStorage/s3Client";
+import { postgresHealthCheck } from "@/app/data/db/postgres_db";
+import { sqlServerHealthCheck } from "@/app/data/db/sqlserver_db";
+
 /**
  * Health check for ECR Viwer
  * @returns Response with status OK.
  */
 export async function GET() {
-  return NextResponse.json({ status: "OK" }, { status: 200 });
+  return NextResponse.json(
+    {
+      status: "UP",
```

Comment:
do we know if anything was checking for the `OK` that needs updating? and/or is this something we need to worry about backwards compat with?

Reply:
No, nothing was checking this value previously.